packages/3d-web-voice-chat/src/chat-ui/components/voice-chat-ui.tsx

```diff
@@ -0,0 +1,114 @@
+import { createRef, forwardRef, useEffect, useRef, useState, MouseEvent } from "react";
+import { flushSync } from "react-dom";
+import { createRoot, Root } from "react-dom/client";
+
+import HourGlass from "../icons/Hourglass.svg";
+import MicrophoneOn from "../icons/MicrophoneOn.svg";
+
+import styles from "./voice-chat-ui.module.css";
+
+type ChatUIInstance = {
+  addMessage: (username: string, message: string) => void;
```

Comment:
This function (and also the related piping and usage) seems unused

Reply:
The entire forward ref (result of using the text chat package as a starting point/boilerplate) was not necessary for this package at all. Fixed on [fifth commit](https://github.com/mml-io/3d-web-experience/pull/42/commits/4552527824b843ae135c4d323ee8fe7571547646)